src/main/kotlin/Main.kt

```diff
@@ -1,36 +1,54 @@
 import java.lang.Math.abs
-import java.lang.Math.random
 import java.util.*
 
 fun main(args: Array<String>) {
-    val r = Random(System.nanoTime()).nextInt(-100, 100)
-    val a = 13
-    val n = IntArray(13)
+    
+    val array = makeArrae()
```

Comment:
Поправь опечатку, плиз
```suggestion
    val array = makeArray()
```